pkgs/desktops/deepin/dde-calendar/default.nix

```diff
@@ -1,8 +1,8 @@
-{ stdenv, fetchFromGitHub, pkgconfig, cmake, qttools,
+{ stdenv, mkDerivation, fetchFromGitHub, pkgconfig, cmake, qttools,
```

Comment:
maybe I'm a bit late here, but why is it called `mkDerivation`? I'd prefer `mkQtDerivation`, to make name distinguishable from stock `mkDerivation`

Reply:
I have same concern about `env` name, which really-really should be called `qtEnv`, but this out of scope of this PR.
cc @ttuegel

Reply:
These names do not exist outside of the scope of Qt packages, so I judge there is little risk of collision. There is a precedent for `mkDerivation` in particular; `haskellPackages` does the same thing.